markets/aave/reservesConfigs.ts

```diff
@@ -138,6 +138,23 @@ export const strategyBAT: IReserveParams = {
   reserveFactor: '2000'
 };
 
+export const strategyCVX: IReserveParams = {
+  optimalUtilizationRate: new BigNumber(0.45).multipliedBy(oneRay).toFixed(),
+  baseVariableBorrowRate: new BigNumber(0).multipliedBy(oneRay).toFixed(),
+  variableRateSlope1: new BigNumber(0.07).multipliedBy(oneRay).toFixed(),
+  variableRateSlope2: new BigNumber(3).multipliedBy(oneRay).toFixed(),
+  stableRateSlope1: new BigNumber(0.1).multipliedBy(oneRay).toFixed(),
+  stableRateSlope2: new BigNumber(3).multipliedBy(oneRay).toFixed(),
+  baseLTVAsCollateral: '4500',
+  liquidationThreshold: '6000',
+  liquidationBonus: '10850',
```

Comment:
LB is not 10%?

Reply:
Gauntlet suggested: "we suggest lowering the liquidation bonus to 8.5% shortly after initial deployment (dependent on market conditions and the results of our economic simulations)" https://governance.aave.com/t/arc-add-support-for-cvx-convex-finance-token/7319/8

So we decided to follow their recommendation

Reply:
Great, thanks